cloudflare/resource_cloudflare_tunnel_route.go

```diff
@@ -25,27 +25,31 @@ func resourceCloudflareTunnelRoute() *schema.Resource {
 
 func resourceCloudflareTunnelRouteRead(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*cloudflare.API)
-
-	tunnelRoute, err := client.GetTunnelRouteForIP(context.Background(), cloudflare.TunnelRoutesForIPParams{
-		AccountID: d.Get("account_id").(string),
-		Network:   d.Get("network").(string),
+	accountID := d.Get("account_id").(string)
+	network := d.Get("network").(string)
+
+	isDeleted := false
+	tunnelRoutes, err := client.ListTunnelRoutes(context.Background(), cloudflare.TunnelRoutesListParams{
+		AccountID:       accountID,
+		IsDeleted:       &isDeleted,
```

Comment:
there is also [`cloudflare.BoolPtr`](https://github.com/cloudflare/cloudflare-go/blob/master/convert_types.go#L75-L86) if you'd like to do this in a single line.